package-lock.json

```diff
@@ -8391,11 +8391,6 @@
         "to-regex-range": "^5.0.1"
       }
     },
-    "filter-obj": {
-      "version": "1.1.0",
-      "resolved": "https://registry.npmjs.org/filter-obj/-/filter-obj-1.1.0.tgz",
-      "integrity": "sha512-8rXg1ZnX7xzy2NGDVkBVaAy+lSlPNwad13BtgSlLuxfIslyt5Vg64U7tFcCt4WS1R0hvtnQybT/IyCkGZ3DpXQ=="
-    },
```

Comment:
@dmytroshch is this generated with node version mentioned in https://bitfinex.slack.com/archives/CKKLH6HA9/p1654509991764169

Reply:
@bhoomij installed and audited with mentioned version